pyuvsim/simsetup.py

```diff
@@ -126,7 +176,38 @@ def _set_lsts_on_uvdata(uv_obj):
 def _create_catalog_diffuse(
     map_nside, diffuse_model, diffuse_params, time, localframe, array_location
 ):
+    """
+    Make a SkyModel object from an analog diffuse map function.
+
+    Only used for internal testing, should not be called by users.
+
+    Parameters
+    ----------
+    map_nside : int
+        HEALPix nside of desired map.
+    diffuse_model : str
+        Name of diffuse model.
+    diffuse_params : dict
+        Dictionary of parameters corresponding to the `diffuse_model`.
+    time : astropy Time object
```

Comment:
autoref `astropy.Time` -- you can add astropy to the `intersphinx_mapping` in the `docs/conf.py` and it will get the relevant link